bin/test-example

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Using env is definitively the favored and most platform independent way. Also it works on Travis without problems.